README.md

```diff
@@ -50,49 +50,64 @@ Sometimes it can be useful to override these options without changing or recompi
 
 Using Maven:
 
-    mvn -Dcucumber.options="..." test
+    mvn -Dcucumber.features="..." -Dcucumber.glue="..." test
 
 Using Gradle:
 
-    gradlew -Dcucumber.options="..." test
-
-Let's look at some things you can do with `cucumber.options`. Try this:
-
-    -Dcucumber.options="--help"
-
-That should list all the available options.
+    gradlew -Dcucumber.features="..." -Dcucumber.glue="..." test
 
 *IMPORTANT*
 
-When you override options with `-Dcucumber.options`, you will completely override whatever options are hard-coded in
+When you override options with `-Dcucumber.<option>`, you will completely override whatever options are hard-coded in
 your `@CucumberOptions` or in the script calling `cucumber.api.cli.Main`. There is one exception to this rule, and that
 is the `--plugin` option. This will not _override_, but _add_ a plugin. The reason for this is to make it easier
 for 3rd party tools to automatically configure additional plugins by appending arguments to a `cucumber.properties`
 file.
 
+| Property name                 | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
This documentation is prone to change and effectively duplicates information from elsewhere. Might be better to use the CLI and allow the user to print the info.

Reply:
Totally agree, I will reference to [this docs](https://github.com/cucumber/cucumber-jvm/tree/master/core#properties-environment-variables-system-options) instead, I didn't know about it earlier.

Reply:
If you can get it to work, it might be preferable to use the maven command. The docs that are kept with the code may not reflect the version used in the skeleton.

Reply:
I resolve this comversation since it's related to the first one